### ISSUE TITLE
Add hotkey instructions to Keyboard Shortcuts dialog

### ISSUE THREAD

**Issue by terracoda** [phetsims/molecules-and-light#327]
With the success of the Pause/Play hotkey in https://github.com/phetsims/scenery-phet/issues/571#, we need to update the Keyboard Shortcuts dialog.

**Comment by terracoda** [comment]
**This is our current keyboard dialog:**
![Screen Shot 2020-04-28 at 11 31 22 AM](https://user-images.githubusercontent.com/1902116/80499887-15866000-8948-11ea-9cc1-acb4e9c59182.png)

@Matthew-Moore240 and @jessegreenberg, what do you think of adding the line:
- Pause or play activated light source [Alt] + [K]

just below the instructions to turn on the light source. This would keep these instructions under the Light Source heading which is kind of nice. The word, "activated" subtly indicates that in order to pause or play, the light source first has to be turned on.

The accessible content for this line would be:
- Pause or play activated light source with Alt key plus K.

We can discuss and today's meeting and adjust the strings as necessary.

I considered calling it a hotkey, but the document itself is called "Keyboard Shortcuts", so maybe that's enough. There is nothing preventing the learner from using the Pause/Play button, itself.

**Comment by terracoda** [comment]
@arouinfar, @BLFiedler and @wandita you are welcome to comment or join our meeting today to comment on the wording of the instructions for the custom shortcut key we have added to MAL.

The shortcut (Alt key + k) is working great in [dev version 1.5.0-dev.47](https://phet-dev.colorado.edu/html/molecules-and-light/1.5.0-dev.47/phet/molecules-and-light_en_phet.html):

**Comment by BLFiedler** [comment]
Should there be any indication that this pauses the entire simulation (inclusive of any interaction process the molecule is undergoing)? Or did you already rule that out? I suspect they would pick up on that fact if the molecule was auditorily wiggling when they paused it and it kept wiggling when they resumed, but maybe not always? 

Another fun interaction I've discovered that I think is a result of tying the photon appearance to the light source toggle: You can make the photons emit faster than what Jesse implemented by rapidly toggling on and off the light source. I discovered this playing with the space/enter key with focus on the light source toggle.

**Comment by terracoda** [comment]
@BLFiedler, that's so funny about the fast firing. It doesn't seem to break the described experience, so that's good.

**Comment by jessegreenberg** [comment]
> Should there be any indication that this pauses the entire simulation

This was my thought too, instead of "light source" maybe say "simulation" since it is pausing/playing light source, photons, and moleculese? 

> Another fun interaction I've discovered that I think is a result of tying the photon appearance to the light source toggle: You can make the photons emit faster than what Jesse implemented by rapidly toggling on and off the light source.

Thanks, this is definitely a bug to solve introduced by reducing the delay of emitting a photon. Ill open a new issue for this. Especially broken when pressing and holding the "enter" key.

**Comment by terracoda** [comment]
I like to use sim language, how about:
- Pause or play action in Observation Window [Alt] + [K]

**Comment by terracoda** [comment]
And the accessible content is:
- Pause or play action in Observation Window with Alt key plus K.

Change "Light Source" heading to "Observation Window"

Decision in meeting Apr. 28, 2020

**Comment by jessegreenberg** [comment]
This content was added, @terracoda can you please review?

**Comment by terracoda** [comment]
Looks good @jessegreenberg, though I think it would be improved if you wrap the text by putting a line break either just before "in Observation Window" or just before "Observation Window".

**Kind of like this:**
![Screen Shot 2020-04-29 at 11 49 34 AM](https://user-images.githubusercontent.com/1902116/80609633-e688f080-8a12-11ea-86f8-c10ec330a53a.png)

**Comment by terracoda** [comment]
In today's interview with our VoiceOver COnsultant we learned that the custom Pause/Play shortcut does not work with Quick Nav enabled. It works fine and as designed when Quick Nav is not on. 

@jessegreenberg, I am assuming that's just how it is. Do you have any comment about this?

**Comment by terracoda** [comment]
@jessegreenberg, I'll move the quick nav comment to a new issue.

**Comment by jessegreenberg** [comment]
Sounds good @terracoda the text now wraps

**Comment by terracoda** [comment]
Looks great, thanks @jessegreenberg 
<img width="899" alt="Screen Shot 2020-05-04 at 9 10 39 PM" src="https://user-images.githubusercontent.com/1902116/81023715-d70f0a80-8e4b-11ea-9cf6-15e80792473c.png">
